cdap-app-templates/cdap-etl/cdap-etl-api/src/main/java/io/cdap/cdap/etl/api/connector/BrowseEntityProperty.java

```diff
@@ -29,9 +29,10 @@
    */
   public enum PropertyType {
     STRING,
-    DATE,
+    DATE_LOCAL_ISO,
```

Comment:
why change the name? (this will break existing classes that use this)

Reply:
This is a request from UI to know exactly what format this date is from the response, I just verified the current migrated plugins do not use this, these are all file related and we do not have them merged yet.